crypto/txsigner.go

```diff
@@ -242,3 +243,15 @@ func encodeSignature(R, S, V *big.Int, isHomestead bool) ([]byte, error) {
 
 	return sig, nil
 }
+
+// Eip3074Magic serialize EIP-3074 messages in form
+// keccak256(type ++ invoker ++ commit)
+func Eip3074Magic(commit []byte, invokerAddr types.Address) []byte {
+	var msg [64]byte
+
+	msg[0] = 0x03
+	copy(msg[13:33], invokerAddr.Bytes())
+	copy(msg[33:], commit)
+
+	return ethgo.Keccak256(msg[:])
```

Comment:
`edge` has already a function to calculate the `keccak` [here](https://github.com/0xPolygon/polygon-edge/blob/develop/crypto/crypto.go#L244).

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1291/commits/1ff84535543bd94f8ac1280842caa8ef3a3133da